conf/maxquant.config

```diff
@@ -34,7 +34,7 @@ params {
     differential_fc_column           = "logFC"
     differential_pval_column         = "P.Value"
     differential_qval_column         = "adj.P.Val"
-    differential_feature_id_column   = "probe_id"
+    differential_feature_id_column   = "Majority protein IDs"
```

Comment:
just want to double check that the previous `probe_id` has now so many different values

Reply:
Yes, this is down to changes by @WackerO that removed some hard-coding on feature ID cols.